validationparser/interceptor.go

```diff
@@ -0,0 +1,46 @@
+package validationparser
```

Comment:
not a great package name. Please refer to [package naming best practices](https://blog.golang.org/package-names)